httpbakery/client.go

```diff
@@ -485,23 +485,31 @@ func postFormJSON(url string, vals url.Values, resp interface{}, postForm func(u
 	return nil
 }
 
-// RequestMacaroons returns any collections of macaroons from the cookies
-// found in the request. By convention, each slice will contain a primary
-// macaroon followed by its discharges.
+// MacaroonsHeader holds the name of the header that can optionally include
```

Comment:
// MacaroonsHeader is the key of the HTTP header
// that can be used to provide a macaroon for request authorization.

?